ship.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  baseBranches: ["maint", "develop"],
+  mergeStrategy: {
+    toSameBranch: ["maint"],
+    toReleaseBranch: {
+      develop: "master"
+    }
+  },
+  buildCommand: () => "yarn build && /bin/bash ./pre-deploy.sh"
```

Comment:
This is from `deploy` script from `package.json`.
Is this all to run before executing `npm publish`?

Reply:
yes, this basically just checks the code in the bundle is correct.
this should probably be an actual test